gradle.properties

```diff
@@ -1,7 +1,7 @@
 org.gradle.caching=true
 org.gradle.jvmargs='-Dfile.encoding=UTF-8'
 group=org.ballerinax.azurefunctions
-version=1.2.37-SNAPSHOT
+version=1.2.39
```

Comment:
Didn't we sync the last release?

Reply:
It was not synced to the 1.2.x branch. So i synced directly with the 1.2.38-stage branch and did the release